BigBaseV2/src/views/vehicle/view_pv.cpp

```diff
@@ -0,0 +1,125 @@
+#include "views/view.hpp"
+#include "fiber_pool.hpp"
+#include "natives.hpp"
+#include "services/mobile/mobile_service.hpp"
+#include "services/vehicle_preview/vehicle_preview_service.hpp"
+#include "util/vehicle.hpp"
+
+namespace big
+{
+	void view::pv() {
+		ImGui::SetWindowSize({ 0.f, (float)*g_pointers->m_resolution_y }, ImGuiCond_Always);
+
+		ImGui::Checkbox("Preview", &g->clone_pv.preview_vehicle);
+		ImGui::SameLine();
+		ImGui::Checkbox("Spawn In", &g->clone_pv.spawn_inside);
+
+		static char plate[9] = { 0 };
+
+		ImGui::Checkbox("Spawn Clone", &g->clone_pv.spawn_clone);
+		if (g->clone_pv.spawn_clone) {
+			ImGui::SameLine();
+			ImGui::Checkbox("Spawn Maxed", &g->clone_pv.spawn_maxed);
+
+			strncpy(plate, g->clone_pv.plate.c_str(), 9);
+			ImGui::Checkbox("Clone PV Plate", &g->clone_pv.clone_plate);
+			if (!g->clone_pv.clone_plate) {
+				ImGui::SetNextItemWidth(300.f);
+				if (ImGui::InputTextWithHint("Plate", "Plate Number", plate, sizeof(plate))) {
```

Comment:
Change this to our text input component, as it currently doesn't disable your keys in game.

Reply:
Done